lib/identity_key/identity_key.c

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <string.h>
+#include <identity_key.h>
+#include <nrf_cc3xx_platform_kmu.h>
+#include <nrf_cc3xx_platform_ctr_drbg.h>
+#include <nrf_cc3xx_platform_identity_key.h>
+#include <hw_unique_key.h>
+#include <zephyr/sys/util.h>
+#include <psa/crypto.h>
+#include <nrfx.h>
+#include <nrfx_nvmc.h>
+
+#include <zephyr/sys/printk.h>
+#include <zephyr/kernel.h>
+
+/* Safe zeroize function from Mbed TLS.
```

Comment:
```suggestion
/* Safe zeroize function inspired by Mbed TLS.
```

Reply:
I removed this completely since I didn't find the reference easily.